package.json

```diff
@@ -263,6 +263,20 @@
           }
         }
       }
+    ],
+    "views": {
+      "explorer": [
+        {
+          "id": "runme.launcher",
+          "name": "Runme Notebooks"
```

Comment:
I believe there's an attribute that controls default visibility (collapsed/uncollapsed). We probably want to decide how front and center we want to be?

Reply:
Added a default collapsed visibility